libs/shared/ui-components/src/Header/HeaderDesktop/HeaderDesktopNavigationProvider/HeaderDesktopDropdown.tsx

```diff
@@ -75,7 +75,7 @@ export const HeaderDesktopDropdown: FC<THeaderDesktopDropdownProps> = ({
         <div className="absolute top-full right-0 z-20 pt-[2.4rem] w-max max-w-[25rem]">
           <ul
             id="options"
-            className="flex flex-col gap-[1.8rem] justify-between items-start p-[2.8rem] min-w-[19.8rem] text-left text-black bg-white/90"
+            className="flex flex-col gap-[1.8rem] justify-between items-start p-[2.8rem] min-w-[19.8rem] text-left text-black bg-white"
```

Comment:
I'm not 100% sure but I strongly suspected that the 90% opacity in the dropdown was creating contrast issues, or strong potential for contrast issues.

Reply:
Looking at it, because the text and menu background are high contrast they do still pass contrast thresholds at a 90% opacity. However, I still don't consider it the most sustainable accessible approach since the changing background below the menu could interfere with it in the future. I prefer not to recommend approaches that put more work on people in the future where possible.

From a design perspective, I'd also advocate for the 100% opacity menu in this case. I can't find less than 100% opacity used almost at all in this website design, so I find full opacity more aligned with the system anyways.